src/script/cache/CacheRepository.ts

```diff
@@ -17,30 +17,34 @@
  *
  */
 
+import {amplify} from 'amplify';
 import {resetStoreValue} from 'Util/StorageUtil';
 
 import {StorageKey} from '../storage/StorageKey';
 
-/**
- * Cache repository for local storage interactions using amplify.
- *
- * @todo We have to come up with a smart solution to handle "amplify.store quota exceeded"
- *  This happened when doing "@cache_repository.set_entity user_et"
- *
- */
 export class CacheRepository {
-  constructor(logger) {
-    this.logger = logger;
+  public static CACHE_KEY = {
+    // Redefine name from "sw.js"
+    ASSETS: `asset: 'asset-cache-v2`,
+  };
+
+  // Caution! When refactoring this code, make sure that it works in an "beforeunload" scenario!
+  static async clearCacheStorage(): Promise<boolean> {
```

Comment:
What do you think of just a module that exports 2 functions instead of a class + static methods?